debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/GenericMongoDbConfigDefTest.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb;
+
+import io.debezium.generic.GenericConfigDefTest;
+
+public class GenericMongoDbConfigDefTest extends GenericConfigDefTest {
```

Comment:
How does this make sure that also the MongoDB connector specific fields are tested? And should there be equivalent classes for the other connectors?

Reply:
It always loads all implementations of `org.apache.kafka.connect.transforms.Transformation`, `org.apache.kafka.connect.transforms.predicates.Predicate` and `io.debezium.config.CommonConnectorConfig` via reflection. So everything visible to the current classloader will be loaded and tested.